src/App.vue

```diff
@@ -1,32 +1,81 @@
 <template>
-  <div id="app">
-    <div id="nav">
-      <router-link to="/">Home</router-link> |
-      <router-link to="/about">About</router-link>
-    </div>
-    <router-view/>
-  </div>
-</template>
+  <v-app>
+    <v-app-bar app dark>
+      <v-toolbar-title>Localizar</v-toolbar-title>
 
-<style>
-#app {
-  font-family: Avenir, Helvetica, Arial, sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-  text-align: center;
-  color: #2c3e50;
-}
+      <v-spacer></v-spacer>
```

Comment:
Taki tip: Jeśli tag nie ma zawartości pomiędzy to można go skrótowo zapisać tak: `<v-spacer />

Reply:
Kiedyś czytałem na jakichś docsach Vue, że zalecają to rozdzielić na otwarcie i zamknięcie taga.